### ISSUE TITLE
Is this repository still maintained?

### ISSUE THREAD

**Issue by brunano21** [rey5137/material#293]

**Comment by 4nd3250n** [comment]
I am asking myself the same question.
It looks like it isn't.

**Comment by manjunathc23** [comment]
:( Damn!

**Comment by MitchTalmadge** [comment]
I believe it is, he is still making commits on different branches. A commit was made on the dev_jcenter branch 19 hours ago: https://github.com/rey5137/material/tree/dev_jcenter and one 4 days ago on the dev_newTab branch: https://github.com/rey5137/material/tree/dev_newTab

**Comment by rey5137** [comment]
Yes, I still maintain it. I'm just too busy with other works currently.
